object_tracker/object_tracker.go

```diff
@@ -128,12 +127,8 @@ func newTracker(ctx context.Context, deps resource.Dependencies, conf resource.C
 
 	// Do the first pass to populate the first set of 2 detections.
 	starterDets := make([][]*track, 2)
-	stream, err := t.cam.Stream(t.cancelContext, nil)
-	if err != nil {
-		return nil, err
-	}
 	for i := 0; i < 2; i++ {
-		img, _, err := stream.Next(t.cancelContext)
+		img, err := camera.DecodeImageFromCamera(cancelableCtx, "", nil, t.cam)
```

Comment:
Thanks for getting rid of the stream.Next(), I forgot we're done with streams in this RDK.  I was gonna say just do t.cam.Image() but this method does that internally and does the decoding back to an image.Image so that's cool.  The only thing I'd say is to pass it a mimeType to try to get back (do utils.MimeTypeJPEG)